.github/ISSUE_TEMPLATE/feature-request.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright The Ratify Authors.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Template created based on verbiage from https://github.com/oras-project/oras/blob/cc40e430ee0cc124250f9855a2e99f535f759221/.github/ISSUE_TEMPLATE/feature-request.yaml
+name: Feature Request
+description: File a feature request
+labels: [enhancement, triage]
+body:
+  - type: markdown
+    id: preface
+    attributes:
+      value: "Thank you for submitting new features for ratify-go."
+  - type: textarea
+    id: description
+    attributes:
+      label: "What would you like to be added?"
+    validations:
+      required: true
+  - type: textarea
+    id: solution
+    attributes:
+      label: "Anything else you would like to add?"
+      description: "Miscellaneous information that will assist in solving the issue."
+    validations:
+      required: false
+  - type: checkboxes
+    id: idea
+    attributes:
+      label: "Are you willing to submit PRs to contribute to this feature?"
+      description: "This is absolutely not required, but we are happy to guide you in the contribution process
+        especially when you already have a good proposal or understanding of how to implement it. Join us at https://slack.cncf.io/ and choose #ratify channel."
+      options:
+        - label: Yes, I am willing to implement it.
```

Comment:
feature request requires design discussion first before any implementation. It is better not to have this checkbox.

Reply:
Updated.